documentation/book/assembly-deployment-configuration-kafka-bridge.adoc

```diff
@@ -0,0 +1,49 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+//attribute for use in snip-external-logging.adoc
+:kind: KafkaBridge
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context-deployment-configuration-kafka-bridge: {context}
+
+[id='assembly-deployment-configuration-kafka-bridge-{context}']
+= Kafka Bridge cluster configuration
+
+:context: deployment-configuration-kafka-bridge
+
+The full schema of the `KafkaBridge` resource is described in the xref:type-KafkaBridge-reference[].
+All labels that are applied to the desired `KafkaBridge` resource will also be applied to the {ProductPlatformName} resources making up the Kafka Bridge cluster.
+This provides a convenient mechanism for resources to be labeled as required.
+
+include::assembly-kafka-bridge-replicas.adoc[leveloffset=+1]
+
+include::assembly-kafka-bridge-bootstrap-servers.adoc[leveloffset=+1]
+
+include::assembly-kafka-bridge-tls.adoc[leveloffset=+1]
+
+include::assembly-kafka-bridge-authentication.adoc[leveloffset=+1]
+
+include::assembly-kafka-bridge-configuration.adoc[leveloffset=+1]
+
+//include::assembly-resource-limits-and-requests.adoc[leveloffset=+1]
+
+//include::assembly-logging.adoc[leveloffset=+1]
+
+include::assembly-healthchecks.adoc[leveloffset=+1]
```

Comment:
Why do we need the healthchecks assembly here?

Reply:
The bridge should have a health-check. It is discussed in https://github.com/strimzi/strimzi-kafka-bridge/pull/211

Reply:
But does this content need to be customised for the Bridge?

Reply:
I think it is. WDYT @ppatierno ?

Reply:
I don't see anything related specifically to bridge in 'Healthcheck configurations' section?

Reply:
Should the 'Configuring healthchecks' procedure also refer to the `KafkaBridge` resource?

Reply:
Yes.